inc/saiswitch.h

```diff
@@ -649,17 +668,19 @@ typedef void (*sai_shutdown_switch_fn)(
  *   via sai_get_switch_attribute().
  *
  * Arguments:
- *   @param[in] profile_id - Handle for the switch profile.
- *   @param[in] switch_hardware_id - Switch hardware ID to open
- *   @param[in] switch_notifications - switch notification table
+ *   @param[out] switch_id - router interface id
+ *   @param[in] attr_count - number of attributes
+ *   @param[in] attr_list - array of attributes
+ *   @param[in] switch_notifications - switch notification table 
  * Return Values:
  *   @return SAI_STATUS_SUCCESS on success
  *           Failure status code on error
  */
 typedef sai_status_t (*sai_connect_switch_fn)(
```

Comment:
we can merge this into create_switch. In create_switch, we can have an CREATE_ONLY attribute named SWITCH_ATTR_INIT_SWITCH whose default value is true, meaning you need to init the switch. However, for connect switch, we can set the attribute to be false. The it won't init the switch.